### PR TITLE
feat: add usePrivacySettings hook

### PR DESCRIPTION
**Pull request by triggertoo**
add hook to get `assetDownloadEnabled` and `assetViewerEnabled` from guideline assets settings

### CODE REVIEW

File: packages/app-bridge/src/react/useGuidelineAssetsSettings.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useState } from 'react';
+import { GuidelineAssetsSettings } from '../types/GuidelineAssetSettings';
+import { Handler } from 'mitt';
+
+export const useGuidelineAssetsSettings = () => {
+    const [assetSettings, setAssetSettings] = useState<GuidelineAssetsSettings>({
```
Comment (silviojaeger) [inline packages/app-bridge/src/react/useGuidelineAssetsSettings.ts:8]: Nit: as we call the hook `useGuidelineAssetsSettings` I would also call the state `assetsSettings`

```suggestion
    const [assetsSettings, setAssetsSettings] = useState<GuidelineAssetsSettings>({
```

File: packages/app-bridge/src/types/Emitter.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ import type { CoverPage } from './CoverPage';
 import type { DocumentGroup } from './DocumentGroup';
 import type { DocumentCategory } from './DocumentCategory';
 import type { BrandportalLink } from './BrandportalLink';
+import { GuidelineAssetsSettings } from './GuidelineAssetSettings';
```
Comment (silviojaeger) [inline packages/app-bridge/src/types/Emitter.ts:14]:
```suggestion
import type { GuidelineAssetsSettings } from './GuidelineAssetSettings';
```

File: packages/app-bridge/src/types/GuidelineAssetSettings.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```
Comment (silviojaeger) [inline packages/app-bridge/src/types/GuidelineAssetSettings.ts:1]: Also here I would name the file `GuidelineAssetsSettings.ts` not `GuidelineAssetSettings.ts`

**Review by silviojaeger — APPROVED** [review]
Left one or two small comments but else looks good👍 Thanks!

File: packages/app-bridge/src/react/useGuidelineAssetsSettings.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import { useEffect, useState } from 'react';
+import { Handler } from 'mitt';
+
+import type { GuidelineAssetsSettings } from '../types/GuidelineAssetsSettings';
+
+export const useGuidelineAssetsSettings = () => {
+    const [assetsSettings, setAssetsSettings] = useState<GuidelineAssetsSettings>({
+        imageAndVideoDownloadEnabled:
+            window.application?.sandbox?.config?.context?.appearance?.privacy?.image.download === 'enable',
+        assetViewerEnabled: !!window.application?.sandbox?.config?.context?.appearance?.asset_viewer_enabled,
```
Comment (SamuelAlev) [inline packages/app-bridge/src/react/useGuidelineAssetsSettings.ts:12]: Move the "getter" to app bridge, the react hook shouldn't know about the implementation of how we get the data

File: packages/app-bridge/src/react/useGuidelineAssetsSettings.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useState } from 'react';
+import { Handler } from 'mitt';
```
Comment (SamuelAlev) [inline packages/app-bridge/src/react/useGuidelineAssetsSettings.ts:4]:
```suggestion
import type { Handler } from 'mitt';
```

File: packages/app-bridge/src/react/useGuidelineAssetsSettings.ts [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+import type { GuidelineAssetsSettings } from '../types/GuidelineAssetsSettings';
+
+export const useGuidelineAssetsSettings = () => {
```
Comment (SamuelAlev) [inline packages/app-bridge/src/react/useGuidelineAssetsSettings.ts:8]:
```suggestion
export const usePrivacySettings = () => {
```
wdyt?

Comment (triggertoo) [reply packages/app-bridge/src/react/useGuidelineAssetsSettings.ts:8]: makes sense 👍

File: packages/app-bridge/src/types/PrivacySettings.ts [hunk 7]
```diff
@@ -0,0 +1,6 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export type PrivacySettings = {
+    assetViewerEnabled?: boolean;
+    imageAndVideoDownloadEnabled?: boolean;
```
Comment (SamuelAlev) [inline packages/app-bridge/src/types/PrivacySettings.ts:5]:
```suggestion
    assetViewerEnabled: boolean;
    assetDownloadEnabled: boolean;
```
`imageAndVideo` => `asset`?
Can you make sure it is defined when you implements the 2 methods? (removed the `?` in the suggestion)

File: packages/app-bridge/src/react/useGuidelineAssetsSettings.ts (hunk 6 above)
Comment (triggertoo) [reply packages/app-bridge/src/react/useGuidelineAssetsSettings.ts]: i'll update the event name's accordingly and the getters in a followup PR in clarify

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**